AirCasting/SearchAndFollow/CompleteScreen/Views/ChartSensor.swift

```diff
@@ -0,0 +1,69 @@
+// Created by Lunar on 23/05/2022.
+//
+
+import Foundation
+import SwiftUI
+
+protocol ChartSensor {
```

Comment:
Not sure this name fits well. Maybe let's rename to `ChartMeasurementsFilter`?

and the `func clean` to func `func filter`?

What do you think?